app/src/main/java/com/google/droidjump/models/LevelManager.java

```diff
@@ -89,12 +87,11 @@ public static LevelStrategy getCurrentLevelStrategy() {
         return gameLevels.get(getCurrentLevelIndex()).getLevelStrategy();
     }
 
-    public static String getCurrentLevelName(){
+    public static String getCurrentLevelName() {
         return gameLevels.get(getCurrentLevelIndex()).getLevelName();
     }
 
-    public static ArrayList<LevelConfig> getGameLevels(){
+    public static ArrayList<LevelConfig> getGameLevels() {
         return (ArrayList<LevelConfig>) gameLevels.clone();
```

Comment:
We should just return List<LevelConfig>.

I am not sure why we are doing a .clone()? Why aren't we just returning the list?

Reply:
Not my code..

Reply:
All code is our team's code. We didn't catch it previously, but now that we have noticed it is better to fix it. 

If you don't want to fix it here as this is not what your PR is about, you can make a separate PR fixing this problem or ask Daria why it is like that - maybe it was actually intended. If not, you could ask her to fix it or fix yourself.

Reply:
We discussed with Daria that she will fix it.